extensions/azurecore/src/account-provider/auths/simpleWebServer.ts

```diff
@@ -0,0 +1,100 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
A simple webserver for handling oauth callbacks.

Separated out for easier testing.